sweepai/events.py

```diff
@@ -152,3 +152,74 @@ class Sender(BaseModel):
     installation: Installation
     repository: Repository
     sender: Sender
+
+
+class ReviewSubmittedRequest(BaseModel):
+    class User(BaseModel):
+        login: str
+        id: int
+        node_id: str
+        avatar_url: str
+        gravatar_id: str
+        url: str
+        html_url: str
+        followers_url: str
+        following_url: str
```

Comment:
Looking at this request, clean up the unused fields:

                request = ReviewSubmittedRequest(**request_dict)
                handle_comment.spawn(
                    repo_full_name=request.repository.full_name,
                    repo_description=request.repository.description,
                    comment=request.review.body,
                    pr_path=None,
                    pr_line_position=None,
                    username=request.review.user.login,
                    installation_id=request.installation.id,
                    pr_number=request.pull_request.number,
                    comment_id=request.review.id,
                )